kong/router/init.lua

```diff
@@ -8,8 +8,121 @@ local kong = kong
 local traditional = require("kong.router.traditional")
 local atc_compat = require("kong.router.atc_compat")
 local utils = require("kong.router.utils")
+local reports = require("kong.reports")
+local semaphore = require("ngx.semaphore")
+local nkeys = require("table.nkeys")
```

Comment:
I think that `nkeys` and `empty_table` should in `phonehome_sync_start do ... end` block